workflow/rules/diffexp.smk

```diff
@@ -262,6 +242,42 @@ rule logcount_matrix:
         "../scripts/sleuth-to-matrix.R"
 
 
+rule get_heatmap:
+    input:
+        Sleuth_logcountmatrix_file="results/tables/logcount-matrix/{model}.logcount-matrix.tsv",
```

Comment:
please adhere to the python style guide (non-capital variable and argument names). Just `logcount_matrix=` is sufficient.

Reply:
Noted